src/js/components/BreadcrumbComponent.jsx

```diff
@@ -12,6 +12,12 @@ var BreadcrumbComponent = React.createClass({
     task: React.PropTypes.string
   },
 
+  shouldComponentUpdate: function (nextProps) {
+    return nextProps.app !== this.props.app
+      || nextProps.group !== this.props.group
```

Comment:
@orlandohohmeier sorry for being the nitpicker here, but as per our word-of-mouth guidelines we settled on trailing `&&` and `||`, not leading ones... :disappointed: